idd/Energy+.idd.in

```diff
@@ -69827,6 +69827,110 @@ SetpointManager:SingleZone:OneStageHeating,
        \required-field
        \type node
 
+SetpointManager:ReturnTemperature:ChilledWater,
+     \memo This setpoint manager is used to place a temperature setpoint on a
+     \memo plant supply outlet node and adjust this setpoint to attempt to achieve a 
+     \memo fixed return water temperature.  The setpoint manager senses plant conditions
+     \memo and adjusts the setpoint each system time step to attempt to achieve this
+     \memo return temperature, which is scheduled.
+     \min-fields 7
+  A1,  \field Name
+       \required-field
+  A2,  \field Plant Loop Supply Outlet Node
+       \note This is the name of the supply outlet node for the plant being controlled by this 
+       \note setpoint manager.  Typically this is where the setpoint will be actuated for
+       \note supply equipment to control to, but not necessarily.  This setpoint manager will
+       \note mine that information from the internal plant data structures.
+       \type node
+       \required-field
+  A3,  \field Plant Loop Supply Inlet Node
+       \note This is the name of the supply inlet node for the plant being controlled with this 
+       \note setpoint manager. The temperature on this node is controlled by actuating the 
+       \note supply setpoint.
+       \type node
+       \required-field
+  N1,  \field Minimum Supply Temperature Setpoint
+       \note This is the minimum chilled water supply temperature setpoint.  This is also used as the default
+       \note setpoint during no-load or negative-load conditions and during initialization.
+       \type real
+       \required-field
+  N2,  \field Maximum Supply Temperature Setpoint
+       \note This is the maximum reset temperature for the chilled water supply.
+       \type real
+       \required-field
+  A4,  \field Return Temperature Setpoint Input Type
+       \note This defines whether the chilled water return temperature target is constant, 
+       \note scheduled, or specified on the supply inlet node by a separate setpoint manager.
+       \type choice
+       \key Constant
+       \key Scheduled
+       \key ReturnTemperatureSetpoint
+       \required-field
+  N3,  \field Return Temperature Setpoint Constant Value
+       \note This is the desired return temperature target, which is met by adjusting the  
+       \note supply temperature setpoint. This constant value is only used if 
+       \note the "Design Chilled Water Return Temperature Input Type" is Constant
```

Comment:
The quotes are not needed on the field name reference, title-case is sufficient.